plugins/cue_points/quiz/lib/api/filters/KalturaQuizUserEntryFilter.php

```diff
@@ -5,4 +5,28 @@
  */
 class KalturaQuizUserEntryFilter extends KalturaQuizUserEntryBaseFilter
 {
+	/**
+	 * @var KalturaNullableBoolean
+	 */
+	public $isAnonymous;
+
+	public function getListResponse(KalturaFilterPager $pager, KalturaDetachedResponseProfile $responseProfile = null)
+	{
+		if(!is_null($this->isAnonymous))
+		{
+			if(KalturaNullableBoolean::toBoolean($this->isAnonymous)===false)
+			{
+				if($this->userIdNotIn==null)
+				{
+					$anonKuserIds = "";
+					$anonKusers = kuserPeer::getKuserByPartnerAndUids(kCurrentContext::getCurrentPartnerId(), array('', 0));
+					foreach ($anonKusers as $anonKuser) {
+						$anonKuserIds .= $anonKuser->getKuserId().",";
```

Comment:
You better append the ids to array and implode it with commas after the array is complete, it will help you to avoid redundant comma at the end of the string.